doc/source/whatsnew/v1.2.0.rst

```diff
@@ -531,6 +531,7 @@ Other
 - Fixed metadata propagation in the :class:`Series.dt` and :class:`Series.str` accessors and :class:`DataFrame.duplicated` and ::class:`DataFrame.stack` methods (:issue:`28283`)
 - Bug in :meth:`Index.union` behaving differently depending on whether operand is a :class:`Index` or other list-like (:issue:`36384`)
 - Passing an array with 2 or more dimensions to the :class:`Series` constructor now raises the more specific ``ValueError``, from a bare ``Exception`` previously (:issue:`35744`)
+- Sorting in descending order being unstable when using :meth:`Series.sort_values` and :meth:`Index.sort_values` for DateTime-like :class:`Index` subclasses (:issue:`35992`)
```

Comment:
can you expand this slightly to say that the sorted results will be different than prior version, I think move this to the API changes section as well.

Reply:
@jreback Sorry, I've expanded the message, but I can't seem to find an API changes section in 1.2.0 whatsnew. Could you point me to where it is in the file?